docs/content/concepts/automation/schedules/troubleshooting.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: "Troubleshooting schedules | Dagster Docs"
+---
+
+# Troubleshooting schedules
+
+Run into issues with a schedule? Use this guide to diagnose and resolve the problem.
+
+---
+
+## Step 1: Verify the schedule is included in the Definitions object
+
+First, verify that the schedule has been included in a <PyObject object="Definitions" /> object. This ensures that the schedule is detectable and loadable by Dagster tools like the Dagster UI and CLI:
+
+```python
+defs = Definitions(
+   assets=[asset_1, asset_2],
+   jobs=[job_1],
+   schedules=[all_assets_job_schedule],
+)
+```
+
+Refer to the [Code locations documentation](/concepts/code-locations) for more info.
+
+---
+
+## Step 2: Verify that the schedule has been started
+
+1. In the Dagster UI, click **Overview > Schedules tab**.
+2. Locate the schedule. Schedules that have been started will have an enabled toggle in the **Running** column:
+
+   <!-- ![Enabled toggle next to a schedule in the Schedules tab of the Overview page](/images/concepts/automation/schedules/schedules-enabled-toggle.png) -->
+
+   <Image
+   alt="Enabled toggle next to a schedule in the Schedules tab of the Overview page"
+   src="/images/concepts/automation/schedules/schedules-enabled-toggle.png"
+   width={1716}
+   height={226}
+   />
+
+---
+
+## Step 3: Verify the schedule's interval configuration
+
+Next, verify that the schedule is using the time interval you expect. In the **Schedules** tab, locate the schedule and look at the **Schedule** column:
+
+<Image
+alt="Highlighted Next tick value for a schedule in the Dagster UI"
+src="/images/concepts/automation/schedules/schedules-next-tick.png"
+width={1716}
+height={226}
+/>
+
+The **Next tick** value indicates when the schedule is next expected to run. In the above image, the next tick is `May 2, 12:00 AM UTC`.
+
+Verify that the time is what you expect, including the timezone.
+
+---
+
+## Step 4: Verify that the UI is using the latest version of the schedule
+
+The next step is to verify that the UI is using the latest version of the schedule code. Use the tabs to view instructions for the version of Dagster you're using.
```

Comment:
The phrasing of "schedule code" here confused me slightly, wondering if it might be the Dagster schedule code?

I wonder if we could say, "make sure the deployed version of your code is up to date", or something along those lines.

Reply:
+1 agreed this is better